README.md

```diff
@@ -21,13 +21,72 @@ when building this AMI.
 
 ## Building the AMI
 
-A Makefile is provided to build the AMI, but it is just a small wrapper around
+A Makefile is provided to build the Amazon EKS Worker AMI, but it is just a small wrapper around
 invoking Packer directly. You can initiate the build process by running the
 following command in the root of this repository:
 
 ```bash
 make
 ```
+By default, Makefile will build Amazon EKS Worker AMI with Kubernetes binaries from these K8s versions
+1.16.15, 1.17.17, 1.18.20, 1.19.14, 1.20.10, 1.21.4
+To build an Amazon EKS Worker AMI for a particular Kubernetes version run the following command
+```bash
+make 1.21.4 ## Build a Amazon EKS Worker AMI for k8s 1.21.4
```

Comment:
I think this should be `make 1.21` ?

In the rest of the doc changes as well, we've been referencing the patch version of the K8s version to build, but we tend to recommend customers only specify the minor version and let the Makefile pick up the latest available patch.

Reply:
Made the change